Patch migrated from desktop gate, where it was listed as:
# date:2010-04-13 owner:wangke type:bug bugid:2986859
Patch3:                 libvisual-03-amd64.diff

which appears to correspond to:
http://sourceforge.net/support/tracker.php?aid=2986859
aka:
https://sourceforge.net/p/libvisual/bugs/13/

which was closed as invalid since upstream converted from autoconf to cmake.

Need to determine if this is still needed with cmake, and if so how to convert
to send upstream.

--- libvisual-0.4.0/configure.ac	2010-04-16 09:56:44.322927624 -0500
+++ libvisual-0.4.0/configure.ac	2010-04-16 09:55:19.191320318 -0500
@@ -35,7 +35,24 @@ case "$host_cpu" in
 	mips)           lv_mips=yes;;
 	alpha)          lv_alpha=yes;;
 	sparc)          lv_sparc=yes;;
-	i*86)           lv_ix86=yes;;
+	i*86)
+          case $host_os in
+            solaris*)
+              AC_CHECK_DECL([__i386], [I386_ABI="yes"], [I386_ABI="no"])
+              AC_CHECK_DECL([__amd64], [AMD64_ABI="yes"], [AMD64_ABI="no"])
+
+              if test "x$I386_ABI" = "xyes" ; then
+                lv_ix86=yes
+              fi
+              if test "x$AMD64_ABI" = "xyes" ; then
+                lv_x86_64=yes
+              fi
+              ;;
+            *)
+              lv_ix86=yes
+              ;;
+          esac
+          ;;
 	x86_64*)	lv_x86_64=yes;;
        	powerpc|ppc)    lv_powerpc=yes;;
 	*)		lv_arch_unknown=yes;;
@@ -780,6 +797,7 @@ echo >>$outfile
    lv_alpha=$lv_alpha
    lv_sparc=$lv_sparc
    lv_ix86=$lv_ix86
+   lv_x86_64=$lv_x86_64
    lv_powerpc=$lv_powerpc
    lv_arch_unknown=$lv_arch_unknown
 
